LogIngestion/azuredeploy.json

```diff
@@ -258,19 +258,13 @@
             "location": {
               "type": "string",
               "defaultValue": "[resourceGroup().location]"
-            },
-            "solutionTypes": {
-              "type": "array",
-              "defaultValue": [
-                "SecurityInsights"
-              ]
             }
           },
           "resources": [
             {
               "type": "Microsoft.OperationalInsights/workspaces",
-              "apiVersion": "2020-10-01",
               "name": "[parameters('workspaceName')]",
+              "apiVersion": "2017-03-15-preview",
```

Comment:
Why do we use the older version in preview ?

Reply:
Good catch. This was by mistake. I reverted this change